### DISCUSSION TITLE
استفاده از اسکریپت برای IP Range های دیگر

### DISCUSSION THREAD

**Discussion by vahidjavaherifar** [MortezaBashsiz/CFScanner#277] [Q&A]
با سلام

میخواستم بدونم امکان استفاده از این اسکریپت برای IP Range های دیگر هم هست؟ مثلا یه Provider دیگه رو بخوایم چک کنیم که IP تمیز داره یا نه؟

**Comment by goingfine** [comment]
نه فقط کلودفلر
